.github/workflows/cherryPick.yml

```diff
@@ -89,7 +86,6 @@ jobs:
 
       - name: Create branch for new pull request
         run: |
-          git config user.name ${{ github.actor }}
```

Comment:
I think this will fail if we don't specify a user name. I think we need to hardcode it to `OSBotify` if we want all CP commits to be done + signed by `OSBotify`:

`git config user.name 'OSBotify'`

Reply:
We set it on line 84: https://github.com/Expensify/App/pull/6655/files#diff-fa867b0856c5b5413c16cafbc0f2308cc061d1f8723c8d71cb40112cf4850c02R84